### ISSUE TITLE
Example clojure provider verification

### ISSUE THREAD

**Issue by benquinteros** [DiUS/pact-jvm#1179]
🙏

**Comment by uglyog** [comment]
Are you using Leiningen? If so see https://github.com/DiUS/pact-jvm/tree/master/provider/lein

Or are you asking for an example of a provider verification test written in Clojure?

**Comment by benquinteros** [comment]
Yeah, an example of a simple provider verification test written in Clojure would be helpful!
